src/components/Card/Card.module.css

```diff
@@ -20,7 +18,6 @@
   background-color: var(--eds-theme-color-background-neutral-default);
 }
 
-
```

Comment:
This and one other spacing change in the PR were automatically made by prettier.